CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -342,74 +342,50 @@ public function import(array $values): void {
       $this->deprecatedFormatParams($paramValues, $formatted);
 
       if ($this->isUpdateExisting()) {
-        //fix for CRM-2219 - Update Contribution
-        // onDuplicate == CRM_Import_Parser::DUPLICATE_UPDATE
-        if (!empty($paramValues['id'])) {
```

Comment:
@eileenmcnaughton just wondering what if a user had chosen to do an update import but only was matching on transaction_id field not on the contribution.id field?

Reply:
@seamuslee001  - it is resolved back here - https://github.com/civicrm/civicrm-core/blob/3a25bebd323c6d459e945ffddcb8a696333e8d04/CRM/Contribute/Import/Parser/Contribution.php#L310